go/stats/opentsdb/collector.go

```diff
@@ -14,151 +14,34 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package opentsdb adds support for pushing stats to opentsdb.
```

Comment:
Moved to `doc.go`

Reply:
Nothing really to see in this file except the addition of `collectOne`. Everything else is just LOC being moved around within the file or to other files.